code/modules/mob/living/carbon/carbon.dm

```diff
@@ -1201,6 +1202,14 @@ so that different stomachs can handle things in different ways VB*/
 		sight |= (SEE_TURFS|SEE_MOBS|SEE_OBJS)
 		see_in_dark = max(see_in_dark, 8)
 
+	if(HAS_TRAIT(src, TRAIT_MESON_VISION))
+		sight |= (SEE_TURFS)
+		lighting_alpha = min(lighting_alpha, LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE)
```

Comment:
No need to have brackets around `SEE_TURFS` here, since it's just the one flag.
```suggestion
	if(HAS_TRAIT(src, TRAIT_MESON_VISION))
		sight |= SEE_TURFS
		lighting_alpha = min(lighting_alpha, LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE)
```